components/AvalancheForecastZoneMap.tsx

```diff
@@ -132,6 +136,27 @@ export const AvalancheForecastZoneMap: React.FunctionComponent<MapProps> = ({cen
     [setPreferences, navigation],
   );
 
+  // Begin Q4 2023 campaign code
+  const [showCampaign, trackCampaign] = useCampaign('campaign-q4-2023', 'map-view');
+  const openCampaignLink = useCallback(() => {
+    trackCampaign();
+    const url = 'https://give.nwac.us/campaign/nwacs-year-end-fundraiser/c536433';
+    Linking.openURL(url).catch((error: Error) => logger.error('Error opening URL', {error, url}));
```

Comment:
Should we use the `Linking` library in the rest of the repo, like on the menu? If so - let's open an issue to track that?

Reply:
I think we do already use it everywhere that's appropriate (places where we want to open the installed browser, as opposed to opening an in-app browser).

Reply:
Ah, makes sense - for my reference, how do we choose which of the two to use?

Reply:
I think we use the Linking library everywhere that makes sense from a product perspective - places where we think we should open an external app, as opposed to showing the page inline in the app. 

The assumption here is that a donation page should probably open in the real browser, so that things like credit card autofill work most effectively.

Reply:
there's probably a fuzzy element to this 😄. I think the product question is do we think this is essentially a "peek" kind of behavior - that the user wants to see something on a single page and then go back to the app? that's a great case for an in-app browser. on the other hand, do we think this might take them off on a series of links? that's probably a good time to open the real browser.

some things, like `mailto:` links, we know *have* to get passed off to the system, so we use `Linking.openURL` for that.

Reply:
Sweet, thanks for explaining that - the "peek" analogy really cleared up how to think about this for me!